scripts/pay-id.v1.yml

```diff
@@ -0,0 +1,38 @@
+info:
+  title: test
+  version: '1.0'
+servers:
+  - url: 'http://localhost:3000'
+paths:
+  '/{path}':
+    parameters:
+      - schema:
+          type: string
+        name: path
+        in: path
+        required: true
+    get:
+      summary: get-pay-id
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Mapping'
+        '404':
+          description: Not Found
+      operationId: resolveKeefer
+      description: Resolve a pay id
+components:
+  schemas:
+    Mapping:
+      title: Mapping
+      type: object
+      properties:
+        address:
+          type: string
+      required:
+        - address
+swagger: "2.0"
```

Comment:
What is the difference between `scripts/pay-id.v1.yml` and `pay-id-api-spec/pay-id.v1.yml`?

Reply:
Removed the former, which was a test to try to debug another alternative build strategy.